.github/workflows/ci.yaml

```diff
@@ -1,4 +1,4 @@
-name: ci
+name: CI
```

Comment:
should we revert back the `Pre-Build` change? 😅

Reply:
Ahaha I'll go ahead and do that!